Makefile

```diff
@@ -139,10 +139,21 @@ $(FFMPEG_ROOT):
 # For ARM64 builds, use the image ghcr.io/viamrobotics/antique:arm64 for backward compatibility
 FFMPEG_PREFIX ?= $(shell realpath .)/gostream/ffmpeg/$(shell uname -s)-$(shell uname -m)
 # See compilation guide here https://trac.ffmpeg.org/wiki/CompilationGuide
-FFMPEG_OPTS = --disable-programs --disable-doc --disable-everything --prefix=$(FFMPEG_PREFIX) --disable-autodetect --disable-x86asm
+FFMPEG_OPTS = \
+	--prefix=$(FFMPEG_PREFIX) \
+	--disable-programs \
+	--disable-doc \
+	--disable-everything \
+	--disable-autodetect \
+	--enable-libx264 \
+	--enable-decoder=h264 \
+	--enable-decoder=mjpeg \
+	--enable-gpl \
+	--enable-demuxer=rtp \
+    --enable-demuxer=rtsp
```

Comment:
Adds RTSP/RTP support. We'll need to run `make ffmpeg` on other platforms to update the static libs.

Reply:
[note] This works well on `aarch64` and `darwin`!